src/attr.rs

```diff
@@ -19,8 +19,24 @@ use crate::{
 pub(crate) fn remove_our_attrs(trait_def: &mut syn::ItemTrait) {
     struct AttrRemover;
     impl VisitMut for AttrRemover {
-        fn visit_trait_item_method_mut(&mut self, m: &mut TraitItemMethod) {
-            m.attrs.retain(|a| !is_our_attr(a));
+        fn visit_trait_item_mut(&mut self, item: &mut TraitItem) {
+            let item_span = item.span();
+            let (attrs, is_method) = match item {
+                TraitItem::Method(m) => (&mut m.attrs, true),
+                TraitItem::Const(c) => (&mut c.attrs, false),
+                TraitItem::Type(t) => (&mut t.attrs, false),
+                TraitItem::Macro(m) => (&mut m.attrs, false),
+                _ => panic!("encountered unexpected `TraitItem`, cannot handle that, sorry!"),
```

Comment:
Guess what will happen if this panic occurs after you emitted an error?

Reply:
Nice catch! However, it is clear that the error handling has to change in some way. Either by using your crate or in another way. But that's not something this PR will fix -- I just applied fixes so that nothing immediately breaks. In short: your comment will be addressed in some future PR.